lib/src/components/month_view_components.dart

```diff
@@ -116,9 +116,6 @@ class FilledCell<T> extends StatelessWidget {
     return Container(
       color: backgroundColor,
       child: Column(
-        mainAxisAlignment: MainAxisAlignment.start,
-        crossAxisAlignment: CrossAxisAlignment.center,
-        mainAxisSize: MainAxisSize.max,
         children: [
```

Comment:
Why we removed `MainAxisSize`, `MainAxisAlignment` and `CrossAxisAlignment`?

Reply:
Values we provided as arguments are default values for Column. So whether we provide it or not doesn't matter.